src/flask/routes/chef.py

```diff
@@ -0,0 +1,62 @@
+import json
+
+from flask import Blueprint, request
+from injector import Injector
+
+from src.api.controllers.chef_controller import ChefController
+from src.constants.audit import ExternalUsers
+from src.core.di_config import DiProviders
+from src.flask.schemas.chef_schema import ChefSchema
+from src.lib.entities.sqlalchemy_orm_mapping import Chef
+
+chef_blueprint = Blueprint("chef_routes", __name__)
+
+ioc = Injector(DiProviders)
+chef_controller = ioc.get(ChefController)
+chef_schema = ChefSchema()
+chef_schemas = ChefSchema(many=True)
+
+
+@chef_blueprint.route("/chefs", methods=["POST"])
+def add():
+    json_chef_data = request.get_json()
+    chef = chef_schema.load(json_chef_data)
+    chef_controller.add(chef)
```

Comment:
Return the value from the controller instead, stop relying on the object references. A references can change at any moment.

Reply:
done.